data.sql

```diff
@@ -118,3 +118,11 @@ VALUES (3, 10, '2020-05-24');
 
 INSERT INTO visits (vets_id, animals_id, date_of_visit)
 VALUES (1, 10, '2021-01-11');
+
+
+
+-- This will add 3.594.280 visits considering you have 10 animals, 4 vets, and it will use around ~87.000 timestamps (~4min approx.)
+INSERT INTO visits (animal_id, vet_id, date_of_visit) SELECT * FROM (SELECT id FROM animals) animal_ids, (SELECT id FROM vets) vets_ids, generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') visit_timestamp;
+
+-- This will add 2.500.000 owners with full_name = 'Owner <X>' and email = 'owner_<X>@email.com' (~2min approx.)
+insert into owners (full_name, email) select 'Owner ' || generate_series(1,2500000), 'owner_' || generate_series(1,2500000) || '@mail.com';
```

Comment:
- [ ] Great work inserting data the way it was recommended. However, I noticed in your screenshots that the execution time before improving performance is significantly lower than **_1 second (1000ms)_**. I would advise you to run this query two or three more times, make the `explain analyze` statement and take screenshots of the execution time that are equal to or more than _**1 second (1000ms)**_. 👍 😎 

#### You can check points _**3 and 4**_ of this [project requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_performance_audit.md) 💻

### Hint: 
- Ensure the queries highlighted below are run before the indexing using the `explain analyze` command: 

  - SELECT COUNT(*) FROM visits where animal_id = 4;

  - SELECT * FROM visits where vet_id = 2;

  - SELECT * FROM owners where email = 'owner_18327@mail.com';

- Then ensure that the execution time for each query is equal to or above **_1 second (1000ms)_** 
- Then take a screenshot and replace the old ones with the new screenshots. 
- Then you apply the indexing and run the `explain analyze` command, take the screenshot and attach the screenshots. 👍 💻